src/modules/renderList.js

```diff
@@ -0,0 +1,60 @@
+import ListElement from './listElement.js';
+
+export default class RenderList {
+  constructor() {
+    this.toDoList = localStorage.getItem('toDoList')
+      ? JSON.parse(localStorage.getItem('toDoList'))
+      : [];
+    this.addTask = this.addTask.bind(this);
+    this.renderList();
+  }
+
+  addTask(description) {
+    const task = new ListElement(this.toDoList.length + 1, description, false);
+    this.toDoList.push(task);
+    this.renderList();
+    this.toLocalStorage();
+  }
+
+  renderList() {
+    const listContainer = document.querySelector('.list__container__checklist');
+    listContainer.innerHTML = '';
+
+    // eslint-disable-next-line no-restricted-syntax
```

Comment:
Kindly avoid disabling linter checks in line. They represent coding standards that you share with your team. You can use a forEach() method here.

Reply:
- [X] forEach method replaced!